packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -93,9 +99,14 @@ class RefreshIndicator extends StatefulWidget {
     this.color,
     this.backgroundColor,
     this.notificationPredicate = defaultScrollNotificationPredicate,
+    this.semanticsEnabled = true,
+    this.semanticsLabel = 'Refresh',
```

Comment:
We'll need to provide a localization for this.

Reply:
This seems like a case then were we definitely want to keep the `semanticsEnabled` flag.

We can't default this to `MaterialLocalizations.of(context)` in the initializer because it's const - we don't have a BuildContext there anyway currently.  I can default it in the `build` method, but then I need to know whether it should be enabled or not independent of nullity of the label.